app/src/main/java/fr/free/nrw/commons/contributions/ContributionController.java

```diff
@@ -307,4 +319,48 @@ private Intent handleImagesPicked(Context context,
         isInAppCameraUpload = false;    // reset the flag for next use
         return shareIntent;
     }
+
+    void getPendingContributions() {
+        final PagedList.Config pagedListConfig =
+            (new PagedList.Config.Builder())
+                .setPrefetchDistance(50)
```

Comment:
I doubt if this is preventing the list from scrolling. I tried with prefetch distance 10 and page size 15. I'm able to scroll through it. Could you please validate the choice of numbers here?

Reply:
I will change the prefetch distance to 10 and page size to 15 in newer commit.

Reply:
It was just a guess. Please validate it before concluding, we must have a reason for the numbers we choose 🙂

Reply:
Sure, I will test out different combinations.

Reply:
But as per the documentation https://developer.android.com/topic/libraries/architecture/paging/data#:~:text=Page%20size%3A%20The%20number%20of,larger%20than%20the%20page%20size. Shouldn't we set the prefetch distance to be much larger value than page size.

Reply:
https://developer.android.com/reference/android/arch/paging/PagedList.Config.Builder

This one says: 
> Should be several times the number of visible items onscreen.

It's not talking about the page size. In any case, both of them are deprecated. Could you check the latest one?

Reply:
Sure I will check the latest one, and update you here

Reply:
Hi Ritika, the guide I sent seems to be for the version Paging 2 which is the same version we are using the app
`PAGING_VERSION=2.1.2` https://github.com/commons-app/apps-android-commons/blob/main/gradle.properties

Reply:
Thanks for checking! While it is not a rule to set one of them greater than the other, the recommendation in the documentation should hold true. Could you check why the recycler view doesn't load more than a specific number of items?

Reply:
Sure, I will check it!

Reply:
I tried uploading 99 contributions which can be seen this video and recycler seems to load all the items. 
https://github.com/user-attachments/assets/e7d084d5-061a-4739-8ce2-1cffa98d2d00

(Please ignore dtest98 which should be dtest99 as I mistakenly wrote dtest46 twice)

Reply:
Working fine in commit [`34930f1`](https://github.com/commons-app/apps-android-commons/pull/5752/commits/34930f1fea1ef8594a01758d3d591b97f8a7c11b)